packages/frontend/ui/src/components/button/button.tsx

```diff
@@ -0,0 +1,31 @@
+import { Pressable } from "pressable";
+import { P } from "typography";
+const variantColors = {
```

Comment:
Is there anyway to type this as valid tailwind css?

Reply:
Not easily... there seem to be ways of doing this, either by string matching some stuff if we know most things are variants of colours, or by using something called sprinkles. I can look into this if we think it would be useful in the long term?

Reply:
(you can quite easily do this with colours)
`const twColors = require("tailwingcss/colors");
type twColor = keyof typeof twColors;`